content/en/blog/community/spinkube-kind-rd/index.md

```diff
@@ -0,0 +1,328 @@
+---
+date: 2024-04-01
+title: "SpinKube on KinD: Rancher Desktop super powers"
+linkTitle: SpinKube on KinD
+description: >
+  Run SpinKube on KinD thanks to Rancher Desktop WASM implementation.
+author: Nuno "the Corsair" Do carmo ([@nunixtech](https://twitter.com/nunixtech))
+resources:
+  - src: "**.{png,jpg}"
+    title: "Image #:counter"
+---
+
+The goal of this guide is show a way to bring [SpinKube](https://www.spinkube.dev/) to [KinD](https://kind.sigs.k8s.io/) without the need of a custom image, like the [SpinKube on k3d](https://www.spinkube.dev/docs/spin-operator/quickstart/) example.
```

Comment:
Just a note to say that kind clusters as of v0.22.0 are ready to host SpinKube and Spin apps using stock Docker (eg without the need for Rancher Desktop).  The one necessary extra step is installation of the kwasm-operator to perform the containerd configuration (eg as shown in the generic ['Installing with Helm' guide](https://www.spinkube.dev/docs/spin-operator/installation/installing-with-helm/#chart-prerequisites)).

That being said, I'm all for providing guides around alternative installation methods like this one :)